package.json

```diff
@@ -46,7 +46,7 @@
     "eth-rpc-errors": "^4.0.0",
     "eth-sig-util": "^3.0.0",
     "ethereumjs-util": "^6.1.0",
-    "ethereumjs-wallet": "0.6.0",
+    "ethereumjs-wallet": "^0.6.4",
```

Comment:
Was this supposed to be updated to v1?

Reply:
no, i prepared an in-range fix https://github.com/ethereumjs/ethereumjs-wallet/commits/v0.6.x
I'd be happy with a bump to v1 but out-of-scope for this